skypy/power_spectrum/_growth.py

```diff
@@ -159,21 +159,18 @@ def integrand(x):
         integrand = (growth_factor(x, cosmology, gamma) - 1) / (1 + x)
         return integrand
 
-    if isinstance(z, int) or isinstance(z, float):
-        integral = integrate.quad(integrand, z, 1100)[0]
-        g = np.exp(integral)
-        growth_function = g / (1 + z)
-
-    elif isinstance(z, np.ndarray):
-        growth_function = np.zeros(np.shape(z))
+    z_flat = np.ravel(z)
```

Comment:
```suggestion
    z_flat = np.ravel(redshift)
```
There is a completely pointless line `z = redshift` above lets delete that and just use the variable `redshift` directly